docs/manual/forms/variables.rst

```diff
@@ -0,0 +1,170 @@
+
+Deze variabelen zijn nog wel beschikbaar, maar we raden aan om deze niet meer te
+gebruiken. In versie 3.0 van Open Formulieren kunnen deze verwijderd worden.
+
+=============== ========= =========================== =========================================================================
+Variabele       Type      Voorbeeldwaarde             Toelichting
+=============== ========= =========================== =========================================================================
+auth.machtigen  object                                Verouderd, wordt in 3.0 verwijderd.
+auth.attribute  string    ``bsn``                     Kan de waarden ``bsn``, ``kvk`` of ``pseudo`` hebben (verouderd,
+                                                      gebruik bij voorkeur ``auth_type``).
+=============== ========= =========================== =========================================================================
+
+.. _manual_forms_variables_auth_context:
+
+Authenticatiecontext
+--------------------
+
+De "authenticatie context" bevat alle beschikbare informatie over *hoe* iemand ingelogd
+is op een formulier. Deze bundel informatie is beschikbaar in de vaste variabele
+``auth_context``.
+
+Wanneer er niet ingelogd is op het formulier, dan is de waarde van deze variabele
+``null``.
+
+.. note:: Het is een nieuwere en volledigere variant van de ``auth`` variabele.
+   Er zijn geen plannen om deze laatste te verwijderen.
+
+De variabele bevat een bak aan informatie, gestructureerd volgens het
+authenticatiecontextdatamodel (TODO: add link). De structuur is als volgt:
+
+.. code-block:: json
+
+    {
+        "source": "string",
+        "levelOfAssurance": "string",
+        "representee": {
+            "identifierType": "string",
+            "identifier": "string"
+        },
+        "authorizee": {
+            "legalSubject": {
+                "identifierType": "string",
+                "identifier": "string"
+            },
+            "actingSubject": {
+                "identifierType": "string",
+                "identifier": "string"
+            }
+        },
+        "mandate": {
+            "role": "string",
+            "services": [
+                {
+                    "id": "string",
+                    "uuid": "string"
+                }
+            ]
+        }
+    }
+
+Merk op dat niet alle attributen aanwezig zijn, dit hangt af van het inlogmiddel (
+DigiD, eHerkenning) en of er wel/niet sprake is van een machtiging én de soort
+machtiging.
+
+De volgende attributen zijn gegarandeerd aanwezig:
+
+* ``source``, mogelijk lege string als waarde
+* ``levelOfAssurance``, mogelijk lege string als waarde
+* ``authorizee``
+* ``authorizee.legalSubject``
+* ``authorizee.legalSubject.identifierType``, mogelijk lege string als waarde
+* ``authorizee.legalSubject.identifier``, mogelijk lege string als waarde
+
+De onderdelen van deze structuur worden ook als individuele variabelen aangeboden:
+
+``auth_context_source``
```

Comment:
I'm not sure if it's practical here (because the descriptions are quite long for some variables), but it might be good to put these in a similar table as the static variables described above. If you glance through these docs it might not be immediately clear that these are extra static variables on top of the ones described in the table

Reply:
I tried putting them in a table initially and then pulled them out because RST tables suck especially with long un-breakable words/code. It makes the actual RST unreadable due to line wrapping/horizontal scrollbars.